en/docs/portal/transports/suspend-activate-transport.md

```diff
@@ -1,10 +1,9 @@
 # Suspend or Activate a transport
-
-The <strong> Transports </strong> page can be reached by clicking "Transports" from the left pane of the HDT portal.
-To suspend or activate a transport, click the "Action" button  of the transport entry in the list and select "Suspend" or "Activate". If the transport is not in the current page, you can search for it in the search box.
+1. Click **Transport** in the left pane to display the Transport page.
```

Comment:
The left pane menu says "Transports" not "Transport".
The page's title is "Transports" not "Transport"